cms/core/tests/test_blocks.py

```diff
@@ -245,7 +246,7 @@ def test_relatedlinksblock__toc(self):
             block.to_table_of_contents_items(block.to_python([])), [{"url": "#related-links", "text": "Related links"}]
         )
 
-    def test_basictableblock_get_context(self):
+    def test_basictableblock__get_context(self):
```

Comment:
Nit: `test_basictableblock__get_context` to `test_basic_table_block__get_context`